tests/Unit/Schema/Directives/HideDirectiveTest.php

```diff
@@ -0,0 +1,92 @@
+<?php declare(strict_types=1);
+
+namespace Tests\Unit\Schema\Directives;
+
+use Tests\TestCase;
+
+class HideDirectiveTest extends TestCase
+{
+    public function testHiddenOnTestingEnv(): void
+    {
+        $this->schema = /** @lang GraphQL */ '
+        type Query {
+            shownField: String! @mock
+            hiddenField: String! @mock @hide(env:["testing"])
+        }
+        ';
+
+        $introspectionQuery = /** @lang GraphQL */ '
+        query  {
```

Comment:
```suggestion
        {
```

Generally omit this, since it is default.